ui/app/templates/components/oidc/assignment-form.hbs

```diff
@@ -1,4 +1,6 @@
-<div>
+{{! TODO remove div, add unless @isInline conditional for header }}
```

Comment:
This comment is complete, yes?

Reply:
I added it just for show when recording the gif - but was waiting until Angel had merged her PR before actually making the change!